fds/services/fds_service.py

```diff
@@ -90,3 +90,36 @@ def commit(self, message: str, yes: bool):
         except Exception as e:
             self.printer.error(str(e))
             raise Exception("Git commit failed to execute")
+
+    def push(self, git_remote: str, dvc_remote: str, ref: str = None):
+        """
+        fds push
+        """
+        try:
+            self.git_service.push(remote=git_remote, ref=ref)
+            self.printer.success("Successfully pushed to Git remote")
+        except Exception as e:
+            self.printer.error(str(e))
+            raise Exception("Git push failed to execute")
+        try:
+            self.dvc_service.push(remote=dvc_remote)
+            self.printer.warn("Successfully pushed to DVC remote")
+        except Exception as e:
+            self.printer.error(str(e))
+            raise Exception("DVC push failed to execute")
+
+    def save(self, message: str, git_remote: str, dvc_remote: str):
+        """
+        fds save
+        """
+        try:
+            self.add(".")
+            self.commit(message)
+            # TODO: add autodetect of remotes, ask users if they want to set a remote,
+            #  and then push to default remotes, instead of manually entering remote names.
+            self.push(git_remote, dvc_remote)
+            self.printer.success("====================================")
+            self.printer.success("Successfully saved current workspace")
+        except Exception as e:
+            self.printer.error(str(e))
+            raise Exception("Git commit failed to execute")
```

Comment:
No need for error handling, already handled by the individual commands
```suggestion
        self.add(".")
        self.commit(message)
        # TODO: add autodetect of remotes, ask users if they want to set a remote,
        #  and then push to default remotes, instead of manually entering remote names.
        self.push(git_remote, dvc_remote)
        self.printer.success("====================================")
        self.printer.success("Successfully saved current workspace")
```